.gitignore

```diff
@@ -0,0 +1,17 @@
+_pycache__/
```

Comment:
You could replace this line with `**/__pycache__/` to ignore `__pycache__` folder across the repository.

Reply:
I have used your recommendations, but it does not work.